.changeset/shy-dots-press.md

```diff
@@ -0,0 +1,5 @@
+---
+"groqd": patch
+---
+
+Adds documentation for select method
```

Comment:
Since the docs are published separately and not released via NPM, I don't think we'll need a changeset here (once we merge this to main, the new docs will be live).

Reply:
oh rad. I'll just remove this then